cmd/mmlogicapi/apisrv/apisrv.go

```diff
@@ -166,7 +165,7 @@ func (s *mmlogicAPI) CreateProposal(c context.Context, prop *mmlogic.MatchObject
 	}
 
 	// Write all non-id fields from the protobuf message to state storage.
-	err := redispb.MarshalToRedis(c, prop, s.pool)
+	err := redispb.MarshalToRedis(c, s.pool, prop, s.cfg.GetInt("redis.expirations.matchobject"))
```

Comment:
A pattern I've noticed from the redispb signatures... by sticking with the wire-format as the data model, any augmentations or decorations to those models going down to the database require parameter-centric signatures. If we end up with more than 4 or 5 parameters, it may be worth investigating a proper datamodel pattern. Cross that bridge when we come to it.